infrastructure/infrastructure.tf

```diff
@@ -75,7 +80,7 @@ module "elastic_resources" {
 }
 
 module "nmap_task" {
-  source = "github.com/ministryofjustice/securityanalytics-taskexecution//infrastructure/ecs_task"
```

Comment:
Do not do this!!!

Reply:
https://www.terraform.io/docs/modules/sources.html#modules-in-package-sub-directories

In fact add a comment so no one else undoes this